packages/fast-components/src/index.ts

```diff
@@ -1,2 +1,2 @@
-export * from "./name-tag";
 export * from "./design-system-provider";
+export * from "./form-associated";
```

Comment:
I don't think we should export this - at least yet - for the reasons I outlined previously.

Reply:
Pulled this export based on our convo